src/module.ts

```diff
@@ -275,44 +275,44 @@ export const enum ExpressionId {
   TableSet = 46 /* _BinaryenTableSetId */,
   TableSize = 47 /* _BinaryenTableSizeId */,
   TableGrow = 48 /* _BinaryenTableGrowId */,
-  Try = 50 /* _BinaryenTryId */,
-  Throw = 51 /* _BinaryenThrowId */,
-  Rethrow = 52 /* _BinaryenRethrowId */,
-  TupleMake = 53 /* _BinaryenTupleMakeId */,
-  TupleExtract = 54 /* _BinaryenTupleExtractId */,
-  RefI31 = 55 /* _BinaryenRefI31Id */,
-  I31Get = 56 /* _BinaryenI31GetId */,
-  CallRef = 57 /* _BinaryenCallRefId */,
-  RefTest = 58 /* _BinaryenRefTestId */,
-  RefCast = 59 /* _BinaryenRefCastId */,
-  BrOn = 60 /* _BinaryenBrOnId */,
-  StructNew = 61 /* _BinaryenStructNewId */,
-  StructGet = 62 /* _BinaryenStructGetId */,
-  StructSet = 63 /* _BinaryenStructSetId */,
-  ArrayNew = 64 /* _BinaryenArrayNewId */,
+  Try = 51 /* _BinaryenTryId */,
```

Comment:
The missing ones here can be found in [this enum](https://github.com/WebAssembly/binaryen/blob/1db044e9654b3147a8d57cc827892d0dcfdd4b85/src/wasm.h#L653) I believe. Should probably add these as well as a `TODO`.